test/Hydration-test.js

```diff
@@ -0,0 +1,70 @@
+import { expect } from 'chai';
+import Hydration from '../src/Hydration';
+
+describe('Hydration', () => {
+  let userData;
+  let repoData;
+  beforeEach(() => {
```

Comment:
beforeEach makes it look so clean - going to see where I can implement this in other test files.